Sources/Sentry/SentryGeo.m

```diff
@@ -0,0 +1,104 @@
+#import "SentryGeo.h"
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation SentryGeo
+
+- (instancetype)init
+{
+    return [super init];
+}
+
+- (id)copyWithZone:(nullable NSZone *)zone
+{
+    SentryGeo *copy = [[SentryGeo allocWithZone:zone] init];
+
+    @synchronized(self) {
+        if (copy != nil) {
+            copy.city = self.city;
+            copy.countryCode = self.countryCode;
+            copy.region = self.region;
+        }
+    }
+
+    return copy;
+}
+
+- (NSDictionary<NSString *, id> *)serialize
+{
+    NSMutableDictionary *serializedData = [[NSMutableDictionary alloc] init];
+
+    @synchronized(self) {
+        [serializedData setValue:self.city forKey:@"city"];
+        [serializedData setValue:self.countryCode forKey:@"country_code"];
+        [serializedData setValue:self.region forKey:@"region"];
```

Comment:
`l`: No need to follow the setValue forKey pattern. I like this syntax more, but it's the same. Up to you.
```suggestion
        serializedData[@"city"] = self.city;
        serializedData[@"country_code"] = self.countryCode;
        serializedData[@"region"] = self.region;
```